README.md

```diff
@@ -12,6 +12,22 @@ API](https://developers.google.com/google-ads/api/docs/start).
 
 ## Requirements
 
+*   Two PHP extensions need to be installed to use the library at runtime
```

Comment:
I see. Thanks for doing research on this.
The suggestion in composer.json looks good to me.
Could you either use do it here or another PR (in case you want to submit this PR quickly)?

Reply:
No worries, I will be sending an update in this PR as I also need to change the default value of the "user C protobuf" argument in the Dockerfile anyway.